packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_InsertGlobalIndices_DP.hpp

```diff
@@ -277,7 +278,7 @@ int executeInsertGlobalIndicesDP_(const comm_ptr_t& comm, const struct CmdLineOp
   {
     // Global assemble the RHS
     TimeMonitor timer(*TimeMonitor::getNewTimer("5) GlobalAssemble (RHS)"));
-    rhs->endFill();
```

Comment:
@csiefer2 Is the intent that users should always call the beginFill and endFill nonmember functions, and never the instance methods?  If so, we could make the one-argument overloads of the nonmember functions friends (defined inline), and make the instance methods private.

Reply:
There's no real harm in calling the members (unless we add code for comm batching).  It just generates more user code.